bundles/org.openhab.core.automation.module.script/src/main/resources/OH-INF/config/script-profile.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="profile:system:script">
+		<parameter name="toItemScript" type="text">
+			<label>To Item Script</label>
+			<description>she Script for transforming states and commands from handler to item.</description>
```

Comment:
```suggestion
			<description>The Script for transforming states and commands from handler to item.</description>
```

Can you also add the default translations to a .properties file for Crowdin?

Reply:
done